torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1147,6 +1147,9 @@ struct PythonPrintImpl {
   void printFunction(
       const Function& func,
       bool print_first_argument_type = true) {
+    if (!func.isGraphFunction()) {
```

Comment:
Serialization will need to unify builtin classes with their corresponding class in the .pt file.  We haven't discussed the expected behavior, but I think we should go further and  _not serialize_ a class that contains builtin methods. We should consider it to be something that can be looked up when read in. Otherwise we get into tricky situations where there is a mix between the already loaded builtin class and what is in the serialized file.